### PR TITLE
Update docs and makefile toml

### PR DESCRIPTION
**Pull request by rustonaut**

### CODE REVIEW

File: Makefile.toml [hunk 1]
```diff
@@ -93,7 +93,7 @@ script = ["""
   fi
 """]
 
-[tasks.run-web]
+[tasks.serve-web]
 script = ["""
```
Comment (acrrd) [inline Makefile.toml:97]: We can remove the `echo warning` now.

File: README.md [hunk 2]
```diff
@@ -169,7 +173,7 @@ python3 -m http.server
 -   path: '../'
 +   git:
 +       url: git@github.com:xaynetwork/xayn_ai_release.git
-+       ref: master
++       ref: staging
```
Comment (acrrd) [inline README.md:176]: If you want to change this put `<branch>`

**Review by acrrd — COMMENTED** [review]
Don't we need to update also the README for instruction to change the html?
Since now we cannot run the threaded version even in chrome we should change the default to sequential also in the `Makefile.toml`.

**Comment by acrrd** [comment]
In the README we also need to change `_baseAssetUrl` from
`https://xayn_ai_staging_assets.s3-de-central.profitbricks.com` to `https://ai-assets.xaynet.dev`.

File: README.md [hunk 3]
```diff
@@ -180,6 +184,12 @@ python3 -m http.server
 +   'https://xayn_ai_staging_assets.s3-de-central.profitbricks.com';
 ```
 
+Then use `cargo make build-web` and `cargo make serve-web` like above.
+
+This only works if the the release repos branch and this branch are in sync
```
Comment (acrrd) [inline README.md:189]: That the interface must mach is usual stuff and for example `flutter build web` gives error if the api we refer to is broken.
Are any case where the mismatch is not visibile at compile time and gives an error at runtime?

Comment (rustonaut) [reply README.md:189]: There could be not all breaking changes are always visible in the API.

Comment (acrrd) [reply README.md:189]: But is not different from any other dependency that we have. We can just add that if there is an unexpected runtime error to first check that the used version is compatible with the example.

**Review by acrrd — APPROVED** [review]

**Pull request merged**